### ISSUE TITLE
Stalker Anomaly - DX10/11 renderers not work

### ISSUE THREAD

**Issue by lufog** [doitsujin/dxvk#4238]
When I try to run Stalker Anomaly

On `DirectX 11`, I get:

![image](https://github.com/user-attachments/assets/84f9f66d-1263-4318-a834-223cda407b38)

On `DirectX 10`, I get to menu, but if I try to start game (or load save) I get stuck with "App is not responding."

The game works fine on `DirectX 9` and `DirectX 8`.

### Software information
S.T.A.L.K.E.R. Anomaly, Gnome 46, Wayland, 64bit wine-prefix, DXVK 2.4 (installed via Winetricks).

### System information
- GPU: NVIDIA GeForce GTX 1660
- Driver: nvidia-open 560.35.03-3
- Wine version: wine 9.16-1
- DXVK version: 2.4 (also tried with daccde7643a2b3ce0e67c2d460d29145c788d6bb)

### Apitrace file(s)
`trace` files are too small, I doubt there will be anything useful in them.

[AnomalyDX10.zip](https://github.com/user-attachments/files/16887064/AnomalyDX10.zip)
[AnomalyDX11.zip](https://github.com/user-attachments/files/16887065/AnomalyDX11.zip)

### Log files
[game_d3d10.log](https://github.com/user-attachments/files/16886410/game_d3d10.log)
[game_d3d11.log](https://github.com/user-attachments/files/16886414/game_d3d11.log)

**Comment by WinterSnowfall** [comment]
d3dx11 (effects library) shouldn't be confused with d3d11. The above will probably work well with the native d3dx11_43.dll or whatnot.

**Comment by lufog** [comment]
@WinterSnowfall, yes, adding this dll fixed the error, the game loads and works. Thank you!
